lib/perfBaseTest.py

```diff
@@ -125,7 +125,7 @@ def tearDown(self):
             desktopHelper.stop_browser(self.browser_type, self.env)
 
         # Delete Url
-        if self.test_url_id:
+        if hasattr(self,"test_url_id"):
```

Comment:
yes, when you use pilot testing (given --sikuli_script when executing runtest.py), there will be no test_url_id assign in test script